docs/sbpy/units.rst

```diff
@@ -43,6 +45,8 @@ sbpy defines two new spectral flux density units: ``VEGA`` and ``JM``.  ``VEGA``
 
 Unit conversions between flux density and Vega-based magnitudes use the `astropy.units equivalency system <https://docs.astropy.org/en/stable/units/equivalencies.html#unit-equivalencies>`_.  sbpy's :func:`~sbpy.units.spectral_density_vega` provides the equivalencies, which astropy would use to convert the units.  The function requires a wavelength, frequency, or bandpass:
 
+.. doctest-requires:: synphot
+
```

Comment:
Do you need this tag before line 62?  I don't know exactly the range of effects for each `..doctest-requires` line, so maybe not?

Reply:
We do.  I fixed this and several other cases.

Reply:
The directive is needed for every code-block that is affected by the presence of whatever packages are given.  There are ways to skip all doctests in a Python source file, but I don't see a way to do it for a whole documentation source file, so I have been marking block-by-block.

Reply:
Thanks to the pytest-doctestplus maintainers, I have learned that whole documentation files can be marked with a package requirement in setup.cfg via:
```
doctest_subpackage_requires =
    docs/sbpy/photometry.rst = astroquery
```

Reply:
Thanks.  I guess we don't have to adopt to this way and clean up all relevant files now, if we want to merge this PR as soon as possible.

Reply:
Yeah, it is just an option we could take.